faktory.go

```diff
@@ -4,5 +4,5 @@ var (
 	Name      = "Faktory"
 	License   = "GPLv3"
 	Licensing = "Licensed under the GNU Public License 3.0"
-	Version   = "0.7.0"
+	Version   = "dev"
```

Comment:
What about binaries using the faktory package on its own, like faktory_worker_go? Ideally they should have a version that they compiled against, right?